server/src/main/java/org/opensearch/search/profile/Profilers.java

```diff
@@ -55,13 +56,15 @@ public final class Profilers {
     public Profilers(ContextIndexSearcher searcher) {
```

Comment:
Please pass the `isConcurrentSegmentSearchEnabled` as the argument to constructor

Reply:
Got it. It has been addressed now.